packages/cicero-core/package.json

```diff
@@ -84,11 +84,13 @@
     "@accordproject/markdown-slate": "0.14.0",
     "@accordproject/markdown-template": "0.14.0",
     "axios": "0.21.1",
+    "chai-fs": "^2.0.0",
```

Comment:
Why do we need this new dependency? Could we not use existing dependencies for this?

Reply:
It's used to check if archive created using toArchive is a file and not empty [example](https://github.com/sanketshevkar/cicero/blob/99d4d09bbcfd8d71c16df1a67840c583f5cd7359/packages/cicero-core/test/template.js#L111). [Chai-fs](https://www.chaijs.com/plugins/chai-fs/) is an official fs plugin for chai library, so I thought it would be a good idea to use it. I can try the test cases using basic fs and chai methods if you want?

Reply:
I've have removed this dependency and simply checking, that the buffer created is not null. I did `npm uninstall --save chai-fs`